monitor_version_scripts/monitor_variables.sh

```diff
@@ -0,0 +1,62 @@
+# #!/bin/bash
+
+apt-get update
+apt-get install -y python3 python3-pip
```

Comment:
nit: python & pip should be available preinstalled on the agents 
also we can install multiple pip packages in one go - `pip install bs4 lxml requests`